src/ui/libs/DatalensChartkit/modules/html-generator/index.ts

```diff
@@ -25,7 +32,7 @@ export function generateHtml(item?: ChartKitHtmlItem | ChartKitHtmlItem[] | stri
             });
         }
 
-        const elem = document.createElement(tag);
+        const elem = document.createElement(tag === TAG_DL_TOOLTIP ? 'div' : tag);
```

Comment:
Is it possible to add the ability to specify a different tag?
When using div, additional styles will be needed when used in the text